Cargo.toml

```
@@ -29,7 +29,7 @@ std = []
 [dependencies]
 alga = { version = "0.9", optional = true }
 cgmath = { version = "0.17", optional = true }
-nalgebra = { version = "0.20", optional = true }
+nalgebra = { version = "0.21", optional = true }
```

Can you bump as a SemVer range, please? :)

```suggestion
nalgebra = { version = ">= 0.20, < 0.22", optional = true }
```

The reason for this is that it will enable more people to use `splines` without having to immediately update to the latest `nalgebra`. It will allow to release this as a minor bump of `splines`.

---

Hmm, after further investigation it looks like nalgebra 0.21 is using simba instead of alga, so I think it will be better to make this a breaking change. Please tell if it's OK for you.

---

I agree. Thanks.